WalletWasabi.Fluent/Styles/Themes/BaseDark.axaml

```diff
@@ -5,6 +5,18 @@
     <Color x:Key="LineChartAreaGradientEndStopColor">#0078A827</Color>
     <Color x:Key="LineChartLineColor">#78A827</Color>
 
+    <Color x:Key="SliderTrackValueFill">#8093AE</Color>
+    <Color x:Key="SliderTrackValueFillPointerOver">#8093AE</Color>
+    <Color x:Key="SliderTrackValueFillPressed">#8093AE</Color>
+    <Color x:Key="SliderThumbBackground">#8093AE</Color>
+    <Color x:Key="SliderThumbBackgroundPointerOver">#8093AE</Color>
+    <Color x:Key="SliderThumbBackgroundPressed">#8093AE</Color>
```

Comment:
I am wondering... There is the `#8093AE` at many places. Wouldn't be simpler if we have one Color and the others just refer to that?
```
<Color x:Key="CoolNameHere">#8093AE</Color>
<StaticResource x:Key="SliderTrackValueFill" ResourceKey="CoolNameHere"/>
```

Also I noticed in Avalonia `SliderTrackValueFill` is defined like this:
```
<StaticResource x:Key="SliderTrackValueFill" ResourceKey="SystemControlHighlightAccentBrush" />
```
It might not work in our cause but overriding `SystemControlHighlightAccentBrush` to `#8093AE` wouldn't be simpler? Or it would modify other colors that we don't want to?

Reply:
> I am wondering... There is the #8093AE at many places. Wouldn't be simpler if we have one Color and the others just refer to that?

No i deliberately made it that way coz there might be a time where we add additional colours for those specific states. 

> It might not work in our cause but overriding SystemControlHighlightAccentBrush to #8093AE wouldn't be simpler? Or it would modify other colors that we don't want to?

That brush is used by alot of controls that we didnt customize. I prefer making things specific so we dont have any unexpected visual changes.

Reply:
I see, thanks!